scripts/update-plugin-list.py

```diff
@@ -0,0 +1,85 @@
+import datetime
+import pathlib
+import re
+
+import packaging.version
+import requests
+import tabulate
+
+FILE_HEAD = r"""Plugins List
+============
+
+PyPI projects that match "pytest-\*" are considered plugins and are listed
+automatically. Packages classified as inactive are excluded.
+"""
+DEVELOPMENT_STATUS_CLASSIFIERS = (
+    "Development Status :: 1 - Planning",
+    "Development Status :: 2 - Pre-Alpha",
+    "Development Status :: 3 - Alpha",
+    "Development Status :: 4 - Beta",
+    "Development Status :: 5 - Production/Stable",
+    "Development Status :: 6 - Mature",
+    "Development Status :: 7 - Inactive",
+)
+
+
+def iter_plugins():
+    regex = r">([\d\w-]*)</a>"
+    response = requests.get("https://pypi.org/simple")
```

Comment:
Never knew such an endpoint exists, which just lists *all* of the PyPI packages. Seems quite non-scalable but if it works, it works :)

Reply:
[The XML-RPC API will be deprecated in the future.](https://warehouse.pypa.io/api-reference/xml-rpc.html) Since i do not know any other way to get a full list from pypi, i used this one. Open for suggestions though!

Why would scalability be an issue down the line?

Reply:
It's perfectly fine, I was just very suprised that that is just one single page which lists all packages in PyPI like this.

> Why would scalability be an issue down the line?

My first thought was to look at npm which is the largest package registry around AFAIK, and indeed: https://blog.npmjs.org/post/157615772423/deprecating-the-all-registry-endpoint

Hopefully we won't see a similar post from the PyPI maintainers any time soon :)